src/state-summary/state-card-script.ts

```diff
@@ -8,6 +8,8 @@ import { isUnavailableState } from "../data/entity";
 import { canRun, ScriptEntity } from "../data/script";
 import { haStyle } from "../resources/styles";
 import { HomeAssistant } from "../types";
+import { computeObjectId } from "../common/entity/compute_object_id";
+import { showMoreInfoDialog } from "../dialogs/more-info/show-ha-more-info-dialog";
 
 @customElement("state-card-script")
```

Comment:
Where will this be used now still? Since `script` is added to `DOMAINS_NO_INFO`? For `group`?

Reply:
I have no idea.

Reply:
There is also still a `<state-card-configurator>` while it is also shown in `DOMAINS_NO_INFO`.